ui/jstests/test_lr_exports.jsx

```diff
@@ -824,7 +824,7 @@ define(['QUnit', 'jquery', 'lr_exports', 'reactaddons',
 
     assert.equal(0, $(container).find("div").size());
     Exports.loader("repo", "user", function() {}, container);
-    assert.equal(1, $(container).find("div").size());
+    assert.equal(2, $(container).find("div").size());
```

Comment:
Where did this change come from? It doesn't look like we are editing `lr_exports.jsx` here